src/test/java/org/takes/facets/auth/PsAllTest.java

```diff
@@ -27,27 +27,29 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
-import org.hamcrest.MatcherAssert;
 import org.hamcrest.core.IsEqual;
 import org.junit.Test;
+import org.llorllale.cactoos.matchers.Assertion;
 import org.takes.Response;
 import org.takes.rq.RqFake;
 import org.takes.rs.RsEmpty;
 
 /**
  * Test of {@link PsAll}.
  * @since 0.22
+ * @checkstyle ClassDataAbstractionCouplingCheck (200 lines)
  */
 public final class PsAllTest {
 
     /**
      * Fails if PsAll with 0 Passes is created.
      * @throws Exception If fails
      */
-    @Test(expected = IllegalArgumentException.class)
+    @Test
     public void thereShouldBeAtLeastOnePass() throws Exception {
-        MatcherAssert.assertThat(
-            new PsAll(
+        new Assertion<>(
+            "An error occurred while executing thereShouldBeAtLeastOnePass()",
```

Comment:
@marceloamadeu Please make this message better, it should show us why the assertion failed.